MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapGestureDetector.java

```diff
@@ -426,8 +426,8 @@ public boolean onFling(MotionEvent e1, MotionEvent e2, float velocityX, float ve
       // tilt results in a bigger translation, limiting input for #5281
       double tilt = transform.getTilt();
       double tiltFactor = 1.5 + ((tilt != 0) ? (tilt / 10) : 0);
-      double offsetX = velocityX / tiltFactor / screenDensity;
-      double offsetY = velocityY / tiltFactor / screenDensity;
+      double offsetX = velocityX / tiltFactor / 3;
```

Comment:
Hm, why actually 3? 🤔 
Will it work OK on non low-density devices?

Reply:
The pixel ratio is usual 2 for low-density devices and 3 for others. Use 3 here the fling offset will be shorter on low-density devices. But let's make it more robust in case there are higher density devices.